### ISSUE TITLE
set `api_version='v1.1'`. The current format will be removed in mem0ai 1.1.0

### ISSUE THREAD

**Issue by nickknyc** [mem0ai/mem0#1731]
### 🐛 Describe the bug

I am getting the following error when trying to do get_all. I have no idea where to set the API version

The current get_all API output format is deprecated. To use the latest format, set `api_version='v1.1'`. The current format will be removed in mem0ai 1.1.0 and later versions.
  print(m.get_all())

**Comment by prateekchhikara** [comment]
Hi @nickknyc, I do not think this was an error, it was supposed to be a warning to upgrade the api_version in the config. Could you please share a screenshot of the error, so that I can pick from there? Thanks

**Comment by nickknyc** [comment]
@prateekchhikara thanks for the quick followup. I figured it out by looking at the graph docs (I am really psyched for the graph feature!!!!) 

Did not understand how to set the api version in the config. I just did the config below and the warning is gone.


config = {
    "llm": {
        "provider": "openai",
        "config": {
            "model": "gpt-4o",
            "temperature": 0.2,
            "max_tokens": 1500,
        }
    },
    "version": "v1.1"
}

**Comment by prateekchhikara** [comment]
Yep, setting the version to "v1.1" in the config will do the work. The default version is "v1.0" which will show the warning shared above.

**Comment by nickknyc** [comment]
i thinj tjis just needs to be covered in the docs and the examples.

**Comment by bruce25796** [comment]
m.api_version = "v1.1"

**Comment by jcaw07** [comment]
I too got this warning. I too could not figure out where to add the version from the docs. I thought I was supposed to add it as a parameter in the m.add() but that was incorrect. But I found this issue, and was able to fix it following the above docs.

[Screenshot 2025-02-19 at 5.40.24 PM.png.pdf](https://github.com/user-attachments/files/18878020/Screenshot.2025-02-19.at.5.40.24.PM.png.pdf)

**Comment by parshvadaftari** [comment]
Closing this as fixed with the latest 1.0.0beta
